ui/views.py

```diff
@@ -280,3 +281,33 @@ def serve_media(request, media_path):
     response['Content-Length'] = os.path.getsize(file_path)
     response['Content-Disposition'] = "attachment; filename=%s" % filename
     return response
+
+
+@login_required
+def serve_resource_exports(request, path):
+    """
+    View to serve media files in case settings.DEFAULT_FILE_STORAGE
+    is django.core.files.storage.FileSystemStorage
+    """
+    media_path = os.path.join(settings.EXPORT_PATH_PREFIX, path)
+    file_path = os.path.join(settings.MEDIA_ROOT, media_path)
+
+    # There is only one path that will work here, make sure it matches exactly.
```

Comment:
Why are we hard-coding a specific path? This prevents a user from doing multiple exports from the same repository. Is it a technical limitation, or a design decision?